javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+window.addEventListener('beforeunload', function (event) {
+  var previousTodosJSON = localStorage.getItem('javascript-local-storage');
```

Comment:
so this functionality needs to be outside of the function, as its job is to retrieve the data from local storage when the app loads